compiler/rustc_erase/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+// This is a separate crate so that we can `allow(incomplete_features)` for just `generic_const_exprs`
+#![feature(generic_const_exprs)]
```

Comment:
Please do not use `generic_const_exprs`, its incredibly incomplete and broken and unsound and will have changes made to it that breaks code that works currently in order to get closer to an actually functioning feature. I do not want to have to deal with `cfg(bootstrap)` pain and working around `generic_const_expr` bugs from usage of the feature _inside of rustc_ whenever this feature gets worked on ^^'

Reply:
I guess to explicitly answer
> for an option on how stable feature(generic_const_exprs) is.

extremely very not stable :sweat_smile: